champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/runes/fortune/FortuneBase.java

```diff
@@ -23,24 +25,30 @@ protected FortuneBase(Champions plugin, String key) {
     @Override
     public List<Component> getRuneLoreDescription(ItemMeta itemMeta) {
         double roll = getRollFromMeta(itemMeta);
-        return List.of(UtilMessage.deserialize("<gray>Fish you catch are <green>%.1f%%<gray> heavier", roll));
+        return new ArrayList<>(Arrays.asList(
+                "<gray>Fish you catch are <green>%.1f%%<gray> heavier",
+                "<gray>Logs you chop have <green>%.1f%%<gray> chance of doubling their drops"
+        )).stream().map(string -> UtilMessage.deserialize(string, roll)).toList();
     }
 
     @Override
     public List<Component> getItemLoreDescription(PersistentDataContainer pdc) {
         int tier = pdc.getOrDefault(RuneNamespacedKeys.TIER, PersistentDataType.INTEGER, 0);
         double roll = pdc.getOrDefault(getAppliedNamespacedKey(), PersistentDataType.DOUBLE, 0.0);
-        return List.of(UtilMessage.deserialize("%s <gray>- Caught fish are <green>%.1f%%<gray> heavier", getStarPrefix(tier), roll));
+        return new ArrayList<>(Arrays.asList(
+                "%s <gray>- Caught fish are <green>%.1f%%<gray> heavier",
+                "%s <gray>- Chopped logs have a <green>%.1f%%<gray> chance of doubling their drops"
+        )).stream().map(string -> UtilMessage.deserialize(string, getStarPrefix(tier), roll)).toList();
```

Comment:
This might be a little weird, seeing caught fish are heavier on an axe
We could maybe look to pass the ItemStack into the getItemLoreDescription function and then just filter what is applied to the item that way